wormhole-connect/src/utils/routes/relay.ts

```diff
@@ -176,10 +176,16 @@ export class RelayRoute extends BridgeRoute {
     routeOptions: RelayOptions,
   ): Promise<number> {
     if (!sendAmount) return 0;
+
+    const { toNativeToken, relayerFee } = routeOptions;
+
+    // floating point math
+    const DECIMALS = 10 ** 8;
```

Comment:
Is this always 8 decimals?

Reply:
This is the token bridge route - which only supports up to 8 decimals right (this serves as an upper bound on # of decimals)

Reply:
ah, I see the division at the bottom (hence why you mentioned "up to").